tests/test_chi_ssa_4.py

```diff
@@ -0,0 +1,134 @@
+import http.client
+
+import pytest  # noqa
+from scrapy import FormRequest
+from scrapy.http import Request, TextResponse
+
+from city_scrapers.spiders.chi_ssa_4 import ChiSsa4Spider
+
+"""
+ChiSsa4Spider can't be tested in the typical way because it does POST
```

Comment:
Here's an example of how to test when parsing from a POST request:

https://github.com/City-Bureau/city-scrapers/blob/9b90ad7ac653c608ccb84b0d0b7f5dc183405b35/tests/test_cook_landbank.py#L11-L37

It shouldn't be overall that different as long as you pull the parsing logic out of functions that perform POST requests. Let me know if the example isn't clear

Reply:
>It shouldn't be overall that different as long as you pull the parsing logic out of functions that perform POST requests. Let me know if the example isn't clear

It seems I should just pull the POST request stuff out of _parse_meetings and we'll be good.